### PR TITLE
Add Galego Language

### PR DESCRIPTION
**Pull request by cyberwichi**

### CODE REVIEW

File: source/CAS/Languages/Galego.php [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+    /**
+     * Get the your were not authenticated string
+     *
+     * @return string not authenticated
+     */
+    public function getYouWereNotAuthenticated()
+    {
+        return '
+        < p>Non estás autenticado.p>< p>Podes volver tentalo facendo  click <a href="%s">aquí</a>.</p>< p>Se o problema persiste debería contactar con el <a href="mailto:%s">administrador deste sitio</a>.</p>';
```
Comment (phy25) [inline source/CAS/Languages/Galego.php:106]: Could you remove the space in the p tag?

File: source/CAS/Languages/Galego.php [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    }
+
+    /**
+     * Get authentication failed string
+     *
+     * @return string authentication failed
+     */
+    public function getAuthenticationFailed()
+    {
+        return 'Autenticación  CAS errada!';
```
Comment (phy25) [inline source/CAS/Languages/Galego.php:95]: Could you remove the duplicate space here and around?

**Review by phy25 — CHANGES_REQUESTED** [review]

File: source/CAS/Languages/Galego.php (hunk 1 above)
Comment (cyberwichi) [reply source/CAS/Languages/Galego.php]: i do it

File: source/CAS/Languages/Galego.php (hunk 2 above)
Comment (cyberwichi) [reply source/CAS/Languages/Galego.php:93]: i do it

Comment (phy25) [reply source/CAS/Languages/Galego.php:93]: There is no change to this line as far as I can tell?

File: source/CAS/Languages/Galego.php [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+    /**
+     * Get the your were not authenticated string
+     *
+     * @return string not authenticated
+     */
+    public function getYouWereNotAuthenticated()
+    {
+        return '
+        <p>Non estás autenticado</p><p>Podes volver tentalo facendo  click <a href="%s">aquí</a>.</p>< p>Se o problema persiste debería contactar con el <a href="mailto:%s">administrador deste sitio</a>.</p>';
```
Comment (phy25) [inline source/CAS/Languages/Galego.php:104]:
```suggestion
        <p>Non estás autenticado</p><p>Podes volver tentalo facendo click <a href="%s">aquí</a>.</p><p>Se o problema persiste debería contactar con el <a href="mailto:%s">administrador deste sitio</a>.</p>';
```

File: source/CAS/Languages/Galego.php [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    }
+
+    /**
+     * Get the service unavailable string
+     *
+     * @return string service unavailable
+     */
+    public function getServiceUnavailable()
+    {
+        return 'O servizo  `<b>%s</b>\' non está dispoñible (<b>%s</b>).';
```
Comment (phy25) [inline source/CAS/Languages/Galego.php:114]:
```suggestion
        return 'O servizo `<b>%s</b>\' non está dispoñible (<b>%s</b>).';
```

File: source/CAS/Languages/Galego.php [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+    }
+
+    /**
+     * Get the should have been redirected string
+     *
+     * @return string should habe been redirected
+     */
+    public function getShouldHaveBeenRedirected()
+    {
+        return 'Xa debería ser  redireccionado ao servidor  CAS. Faga  click <a href="%s">aquí</a> para continuar';
```
Comment (phy25) [inline source/CAS/Languages/Galego.php:83]:
```suggestion
        return 'Xa debería ser redireccionado ao servidor CAS. Faga click <a href="%s">aquí</a> para continuar';
```

File: source/CAS/Languages/Galego.php [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+    }
+
+    /**
+     * Get logout string
+     *
+     * @return string logout
+     */
+    public function getLogout()
+    {
+        return 'Saída  CAS necesaria!';
```
Comment (phy25) [inline source/CAS/Languages/Galego.php:73]:
```suggestion
        return 'Saída CAS necesaria!';
```

File: source/CAS/Languages/Galego.php [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+    }
+
+    /**
+     * Get authentication wanted string
+     *
+     * @return string authentication wanted
+     */
+    public function getAuthenticationWanted()
+    {
+        return 'Autenticación  CAS necesaria!';
```
Comment (phy25) [inline source/CAS/Languages/Galego.php:63]:
```suggestion
        return 'Autenticación CAS necesaria!';
```

File: source/CAS/Languages/Galego.php [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+    }
+
+    /**
+     * Get authentication failed string
+     *
+     * @return string authentication failed
+     */
+    public function getAuthenticationFailed()
+    {
+        return 'Autenticación  CAS errada!';
```
Comment (phy25) [inline source/CAS/Languages/Galego.php:93]:
```suggestion
        return 'Autenticación CAS errada!';
```

**Review by phy25 — CHANGES_REQUESTED** [review]

**Review by phy25 — CHANGES_REQUESTED** [review]
Could you force-push your branch without commits of those irrevelant files, or create a new PR without those files?

**Pull request closed without merging**